tasks/task05/task05.py

```diff
@@ -0,0 +1,34 @@
+def create_user(name, surname, **kwargs) -> dict:
+    res = {name: name, surname: surname, "age": 42, "extra": {}}
+    for key, value in kwargs.items():
+        if key == 'age':
+            res[key] = value
+        else:
+            res["extra"][key] = value
```

Comment:
in case you have `age` as a separate keyword argument, you do not  have to loop over `kwargs.items` and can just assign `kwargs` to `extra`